.changeset/happy-hounds-hope.md

```diff
@@ -0,0 +1,51 @@
+---
+'@leafygreen-ui/a11y': patch
+'@leafygreen-ui/badge': patch
+'@leafygreen-ui/banner': patch
+'@leafygreen-ui/box': patch
+'@leafygreen-ui/button': patch
+'@leafygreen-ui/callout': patch
+'@leafygreen-ui/card': patch
+'@leafygreen-ui/checkbox': patch
+'@leafygreen-ui/code': patch
+'@leafygreen-ui/combobox': patch
+'@leafygreen-ui/confirmation-modal': patch
+'@leafygreen-ui/copyable': patch
+'@leafygreen-ui/emotion': patch
+'@leafygreen-ui/expandable-card': patch
+'@leafygreen-ui/form-footer': patch
+'@leafygreen-ui/guide-cue': patch
+'@leafygreen-ui/hooks': patch
+'@leafygreen-ui/icon': patch
+'@leafygreen-ui/icon-button': patch
+'@leafygreen-ui/inline-definition': patch
+'@leafygreen-ui/interaction-ring': patch
+'@leafygreen-ui/leafygreen-provider': patch
+'@leafygreen-ui/lib': patch
+'@leafygreen-ui/logo': patch
+'@leafygreen-ui/marketing-modal': patch
+'@leafygreen-ui/menu': patch
+'@leafygreen-ui/modal': patch
+'@leafygreen-ui/palette': patch
+'@leafygreen-ui/pipeline': patch
+'@leafygreen-ui/popover': patch
+'@leafygreen-ui/portal': patch
+'@leafygreen-ui/radio-box-group': patch
+'@leafygreen-ui/radio-group': patch
+'@leafygreen-ui/ripple': patch
+'@leafygreen-ui/segmented-control': patch
+'@leafygreen-ui/select': patch
+'@leafygreen-ui/side-nav': patch
+'@leafygreen-ui/stepper': patch
+'@leafygreen-ui/table': patch
+'@leafygreen-ui/tabs': patch
+'@leafygreen-ui/text-area': patch
+'@leafygreen-ui/text-input': patch
+'@leafygreen-ui/toast': patch
+'@leafygreen-ui/toggle': patch
+'@leafygreen-ui/tokens': patch
+'@leafygreen-ui/tooltip': patch
+'@leafygreen-ui/typography': patch
+---
+
+- Updates the build to remove an implicit dependency on `@emotion/react`. LG packages will not build if `@leafygreen/emotion@4.0.2` or greater is installed since this package has since removed `@emotion/react` as a dependency.
```

Comment:
Can we update this wording to not sound like a breaking change? It sounds like users of the library can't use 4.0.2 or greater with this change.

A good fix would be to say:
"Fixes an issue where LeafyGreen packages would not build if `@leafygreen/emotion@4.0.2` or greater is installed by removing an implicit build dependency on `@emotion/react`.